### DISCUSSION TITLE
Implement "ILIKE" or something similar on the search bar

### DISCUSSION THREAD

**Discussion by Fred-A-P** [outline/outline#5027] [Ideas]
Sometimes I have some trouble finding stuff in outline, I never knew why exactlly.

Today I found a good example of something that can improve the experience a bit.

Implementing something to find similar words would help when there is some kind of grammar error or mistyping.

Here is an example where a grammar error prevented me from seeing a specific document:

![image](https://user-images.githubusercontent.com/62549035/224714856-826c9ccb-81f5-4a75-9799-1cae11d12e97.png)

**Comment by tommoor** [comment]
Interesting, as an English speaker/writer this particular problem hasn't hit me before. We could certainly improve the handling of accents in the search index (essentially mapping them to the closest character) in both directions
